src/state-summary/state-card-input_number.html

```diff
@@ -41,14 +57,25 @@
         hidden='[[hiddenbox]]'
       >
       </paper-input>
+      <div class="state" hidden='[[hiddenbox]]'>[[stateObj.attributes.unit_of_measurement]]</div>
+      <div id="sliderstate" class="state sliderstate" hidden='[[hiddenslider]]'>[[value]] [[stateObj.attributes.unit_of_measurement]]</div>
     </div>
   </template>
 </dom-module>
 
 <script>
-class StateCardInputNumber extends Polymer.Element {
+class StateCardInputNumber extends Polymer.mixinBehaviors([
+  Polymer.IronResizableBehavior
+], Polymer.Element) {
   static get is() { return 'state-card-input_number'; }
 
+  ready() {
+    super.ready();
+    if (this.mode === 'slider') {
```

Comment:
mode can change later, so it shouldn't be done once

Reply:
Fixed in https://github.com/home-assistant/home-assistant-polymer/pull/808/commits/9fcab8681a5de1ef8e3093374e074db04709954a